RecipeApp/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,114 @@
+// ProfileViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Протокол для вью
+protocol ProfileView: AnyObject {
+    func updateView(model: ProfileModel)
+}
+
+/// Экран для профиля
+final class ProfileViewController: UIViewController, ProfileView {
+    // MARK: - Properties
+
+    var presenter: ProfilePresenter?
+    private var activities = ["bonuses", "terms & privacy policy", "log out"]
```

Comment:
'n
```suggestion
    private var activities = ["bonuses", "terms & privacy policy", "log out"]
```
эта история из Презентера должна была приехать